app/v1/services/RetrieveOtherReliefsService.scala

```diff
@@ -40,19 +41,28 @@ class RetrieveOtherReliefsService @Inject() (connector: RetrieveOtherReliefsConn
       correlationId: String): Future[ServiceOutcome[RetrieveOtherReliefsResponse]] = {
 
     val result = for {
-      responseWrapper <- EitherT(connector.retrieve(request)).leftMap(mapDownstreamErrors(desErrorMap))
+      responseWrapper <- EitherT(connector.retrieve(request)).leftMap(mapDownstreamErrors(errorMap))
     } yield responseWrapper
 
     result.value
   }
 
-  private def desErrorMap =
-    Map(
+  private val errorMap = {
+    val errors = Map(
       "INVALID_TAXABLE_ENTITY_ID" -> NinoFormatError,
       "FORMAT_TAX_YEAR"           -> TaxYearFormatError,
       "NO_DATA_FOUND"             -> NotFoundError,
+      "INVALID_CORRELATIONID"     -> InternalError,
       "SERVER_ERROR"              -> InternalError,
       "SERVICE_UNAVAILABLE"       -> InternalError
     )
 
+    val extraTysErrors = Map(
+      "TAX_YEAR_NOT_SUPPORTED" -> RuleTaxYearNotSupportedError,
+      "INVALID_CORRELATION_ID" -> InternalError
```

Comment:
missing `INVALID_TAX_YEAR -> TaxYearFormatError`

Reply:
Looks like this was a bug in the pre-TYS implementation: FORMAT_TAX_YEAR should be INVALID_TAX_YEAR.